app/src/lib/rules/rules_processor.test.js

```diff
@@ -1,7 +1,8 @@
 import {TOKEN_TYPE, create_clause_token, create_token, flatten_sentence} from '$lib/parser/token'
 import {describe, expect, test} from 'vitest'
 import {parse_checker_rule, parse_transform_rule} from './rules_parser'
-import {apply_checker_rules, apply_transform_rules} from './rules_processor'
+import {apply_rules} from './rules_processor'
+import { LOOKUP_RULES } from './lookup_rules'
```

Comment:
```suggestion
import {LOOKUP_RULES} from './lookup_rules'
```

Reply:
I guess the linter doesn't catch that one!